src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -183,13 +184,13 @@ public void getUsers(String query, String orderBy,
               try {
                 response.write(JSON_MAPPER.writeValueAsString(user), "UTF-8");
               } catch (JsonProcessingException e) {
-                logger.error("This error should never happen: " + e);
+                throw new UncheckedIOException(e);
               }
             }, reply -> {
               if (reply.succeeded()) {
                 response.write(String.format(JSON_USERS_FOOTER, cnt[0], cnt[0]));
               } else {
-                response.setStatusCode(500).setStatusMessage(reply.cause().getLocalizedMessage());
+                response.setStatusCode(500).setStatusMessage(reply.cause().getMessage());
```

Comment:
I noticed that in streaming mode, once we start to write response body, we can not change response status code anymore. HTTP/2 can do that, but we are not there yet. So the best we can do for now is to write the error message to the response body. So this line needs a bit tweak.

Reply:
Yes. I've created https://issues.folio.org/browse/MODUSERS-169